pyscriptjs/src/pyodide.ts

```diff
@@ -97,6 +97,32 @@ export class PyodideRuntime extends Runtime {
         }
     }
 
+    /**
+     *
+     * @param path : the path in the filesystem
+     * @param fetch_path : the path to be fetched
+     *
+     * For a given resource with `fetch_path` as `http://dummy.com/hi.py`
+     * and the location of `path` in the FS to be `a/b/c/foo.py`
```

Comment:
This reads a bit confusing to me. A good best practice is to start with a good summary sentence of what the function does. Something like  "Given a file available at a `fetch_path` URL (i.e. `http://dummy.com/hi.py`), downloads the such file and saves to the `path` on the FS."

Reply:
Thanks, changed a bit!